docs/docs/recipes/gitlab-continuous-integration.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Recipes: Continuous Integration on Gitlab"
+tableOfContentsDepth: 1
+---
+
+Push your code to gitlab and automate your production build!
```

Comment:
Can we get a bit of an intro to this? See the intro of https://www.gatsbyjs.org/docs/recipes/working-with-themes#creating-a-new-site-using-a-theme for context. 
```suggestion
Push your code to GitLab and automate your production build!
```